docroot/themes/humsci/humsci_basic/src/scss/partials/admin/_user-login-page.scss

```
@@ -0,0 +1,179 @@
+main:has(.user-login-form) {
+  width: 100% !important;
+  max-width: 100% !important;
+  height: 100%;
+  min-height: 100vh; 
+  padding: hb-calculate-rems(128px) 0;
+  background-image: url("/themes/humsci/humsci_basic/images/user-login-background-image.png");
+  background-position: center;
+  background-repeat: no-repeat;
+
+  .block--page-title {
+    display: none;
+  }
+}
+
+.user-login-form {
+  background-color: var(--palette--white);
+  padding: hb-calculate-rems(24px);
+  max-width: hb-calculate-rems(900px);
+  width: auto;
+  margin: hb-calculate-rems(24px);
+  margin-bottom: hb-calculate-rems(24px) !important;
+
+  @include grid-media-min('md') {
+    width: fit-content;
+    margin: 0 auto;
+    margin-bottom: 0 !important;
+    padding: hb-calculate-rems(24px) hb-calculate-rems(32px);
+  }
+
+  @include grid-media-min('lg') {
+    padding: hb-calculate-rems(24px) hb-calculate-rems(48px);
+  }
+
+  div[data-drupal-selector="edit-intro-text"] h1 {
+    display: block !important;
```

`important` is not necessary here.

```suggestion
    display: block;
```